sw/device/silicon_creator/lib/ownership/keys/fake/BUILD

```diff
@@ -183,3 +184,27 @@ pkg_filegroup(
     ],
     prefix = "fake_keys",
 )
+
+keyset(
+    name = "ecdsa_keyset",
+    build_setting_default = "",
+    keys = {
+        "app_prod_ecdsa_p256.der": "prod_key_0",
+        "app_dev_ecdsa_p256.der": "dev_key_0",
+        "app_test_ecdsa_p256.der": "test_key_0",
+        "app_unauthorized_ecdsa_p256.der": "unauthorized_key_0",
+    },
+    profile = "local",
+    tool = "//signing/tokens:local",
+)
+
+keyset(
+    name = "spx_keyset",
+    build_setting_default = "",
+    keys = {
+        "app_prod_spx.pem": "prod_key_0",
+        "app_dev_spx.pem": "dev_key_0:domain=PreHashedSha256",
```

Comment:
I think the nicknames for the application keys should include `app`.  E.g. `app_prod_0`, `app_test_0`, `app_dev_0`.